src/core_ocean/shared/mpas_ocn_tendency.F

```diff
@@ -326,6 +330,17 @@ subroutine ocn_tend_vel(tendPool, statePool, forcingPool, diagnosticsPool, meshP
       !
       call ocn_vel_vadv_tend(meshPool, normalVelocity, layerThicknessEdge, vertAleTransportTop, tend_normalVelocity, err)
 
+      ! Compute tidal potential forcing, if needed 
+      if (config_use_tidal_potential_forcing) then
+        call ocn_compute_tidal_potential_forcing(meshPool, forcingPool, diagnosticsPool, err)
+      end if
+
+      if (config_use_tidal_potential_forcing .and. config_time_integrator == 'RK4') then
```

Comment:
Should this work in split explicit, too?  Is the RK4 flag needed?

Reply:
For split explicit, we only want to add the tidal potential to the barotropic equations.

Reply:
Really?  I'm surprised by that.

Reply:
@mark-petersen and @pwolfram, can you confirm that?  I don't see why the pressure gradient should be computed differently between the 2 time integration approaches.

Reply:
I thought I understood clearly yesterday when we discussed @mark-petersen and @sbrus89 but not I'm not so sure.  Can we please clarify here for posterity?  

My recollection is that for split explicit the pressure gradient term already has the barotropic contribution removed, but for split explicit the baroclinic and barotropic are combined, hence for split explicit the calls to `ocn_vel_pressure_grad_tend` only worry about the baroclinic terms (no need for adjustment of the tidal potential) whereas for RK4 they include both baroclinic and barotropic (and hence the need for adjustment of the tidal potential).  Thus in the split explicit routine there is the tidal potential, and because there is no equivalent change in the RK4 routine, it is made in the tendency and hence the need for the RK4 flag.  Thus, the application of the tidal potential forcing is different in each approach and hence the code doesn't "overlap" for each case.

Would you both concur this is the correct interpretation or have I missed something?  Just want to make sure we have this written down clearly so we are all on the same page.

Reply:
@pwolfram, that is correct as far as I understand it.